content/company/clients.mdx

```diff
@@ -568,7 +568,7 @@ _body:
           solution  and bug fixing existing errors on their existing site. SSW
           was also  commissioned to take a new design from eMove and create and
           implement  a DotNetNuke skin for the new design.
-        caseStudyUrl: 'https://www.ssw.com.au/SSW/Consulting/Case-Study/eMove.aspx'
+        caseStudyUrl: '/company/clients/e-move'
```

Comment:
I think route should be `emove`, it's a single word

Reply:
Done